docs/index.html

```diff
@@ -303,7 +317,7 @@
                 "ring": COMPETENT,
                 "label": "Laravel",
                 "active": true,
-                "moved": 1
+                "moved": 0
             },
             {
                 "quadrant": PLATFORM,
```

Comment:
I think this can be removed

Reply:
Remove laravel? We use this for both studiekeuze and newer wordpress sites

Reply:
It was about Svelte. The line under laravel.

Reply:
Ah 👍 
And your other comments, are they about Google Analytics and Typeform or about other tech?

Reply:
yes, same applies for those comments.